client/src/components/ViewModeToggle/ViewModeToggle.scss

```diff
@@ -0,0 +1,122 @@
+.viewmode-dropdown-toggle {
+  margin-left: auto;
+  font-size: 13px;
+
+  #splitModeButton {
+    &:disabled {
+      pointer-events: all;
+    }
+  }
+
+  .viewModeToggle-chosenView-title {
+    font-size: 10px;
+    line-height: 10px;
+    vertical-align: top;
+    white-space: normal;
+    width: 44px;
+    display: inline-block;
+    text-align: left;
+  }
+}
+
+.dropdown {
+
+  &.viewModeToggle {
+    position: absolute;
+    right: 10px;
+
+    .viewmode-dropdown-toggle {
+      &:before {
+        font-size: 18px;
```

Comment:
`$h3-font-size: 17px` or `$h2-font-size: 19px` - if you could use one of those we'd be reducing customisation away from core